SITnett/graphics/style.css

```diff
@@ -187,14 +187,14 @@ content {
 	display: grid;
 	max-width: 1200px;
 	margin: auto;
-	grid-template-columns: repeat(auto-fit, minmax(120px, 1fr));
+	grid-template-columns: repeat(auto-fill, minmax(120px, 1fr));
     grid-auto-rows: min-content;
     grid-gap: 1.0rem 0.5rem;
     justify-content: center;
 }
 
 .production-wrapper .card-wrapper{
-	grid-template-columns: repeat(auto-fit, minmax(280px, 1fr));
+	grid-template-columns: repeat(auto-fill, minmax(280px, 1fr));
```

Comment:
Endringen her er ikke relevant for nummerlisten: Den endrer oppførselen hvor kort som står ensom på en linje fyller hele linjen, til å heller følge kolonnestørrelsene:

<img width="946" alt="bilde" src="https://user-images.githubusercontent.com/16143117/214373409-72194eed-d48e-4bc5-9f3b-eee22f17fb29.png">